doc/sphinx/changes.rst

```diff
@@ -2,6 +2,441 @@
+  In particular Coq is now more aggressive in unfolding constants
+  when it looks for a iota redex. Performance regression may occur
+  in Fixpoint declarations without an explicit {struct} annotation,
+  since guessing the decreasing argument can now be more expensive.
+  (PR #9602)
+
+Notations
+
+- New command `Declare Scope` to explicitly declare a scope name
+  before any use of it. Implicit declaration of a scope at the time of
+  `Bind Scope`, `Delimit Scope`, `Undelimit Scope`, or `Notation` is
+  deprecated.
+
+- New command `String Notation` to register string syntax for custom
+  inductive types.
+
+- Numeral notations now parse decimal constants such as 1.02e+01 or
+  10.2. Parsers added for Q and R. This should be considered as an
+  experimental feature currently.
+  Note: in -- the rare -- case when such numeral notations were used
+  in a development along with Q or R, they may have to be removed or
+  deconflicted through explicit scope annotations (1.23%Q,
+  45.6%R,...).
+
+- Various bugs have been fixed (e.g. PR #9214 on removing spurious
+  parentheses on abbreviations shortening a strict prefix of an application).
+
+- Numeral Notations now support inductive types in the input to
+  printing functions (e.g., numeral notations can be defined for terms
+  containing things like `@cons nat O O`), and parsing functions now
+  fully normalize terms including parameters of constructors (so that,
+  e.g., a numeral notation whose parsing function outputs a proof of
+  `Nat.gcd x y = 1` will no longer fail to parse due to containing the
+  constant `Nat.gcd` in the parameter-argument of `eq_refl`).  See
+  #9840 for more details.
+
+- Deprecated compatibility notations have actually been removed.  Uses
+  of these notations are generally easy to fix thanks to the hint
+  contained in the deprecation warnings. For projects that require
+  more than a handful of such fixes, there is [a
+  script](https://gist.github.com/JasonGross/9770653967de3679d131c59d42de6d17#file-replace-notations-py)
+  that will do it automatically, using the output of coqc. The script
+  contains documentation on its usage in a comment at the top.
+
+Plugins
+
+- The quote plugin (https://coq.inria.fr/distrib/V8.8.1/refman/proof-engine/detailed-tactic-examples.html#quote)
+  was removed. If some users are interested in maintaining this plugin
+  externally, the Coq development team can provide assistance for extracting
+  the plugin and setting up a new repository.
+
+Tactics
+
+- Removed the deprecated `romega` tactics.
+- Tactic names are no longer allowed to clash, even if they are not defined in
+  the same section. For example, the following is no longer accepted:
+  `Ltac foo := idtac. Section S. Ltac foo := fail. End S.`
+
+- The tactics 'lia','nia','lra','nra' are now using a novel
+  Simplex-based proof engine. In case of regression, 'Unset Simplex'
+  to get the venerable Fourier-based engine.
```

Comment:
If we decide to have a "Main changes" / "Other changes" separation, then the note about `Unset Simplex` should move to the previous section.